Golem.Tests/Utils.cs

```diff
@@ -10,6 +10,7 @@
 
 using Microsoft.Extensions.Logging;
 using System.Runtime.InteropServices;
+using ICSharpCode.SharpZipLib.Zip;
```

Comment:
Is there any functionality that needs this `using`?

Reply:
Removed